pyabc/sampler/base.py

```diff
@@ -34,6 +34,23 @@ def all_sum_stats(self):
         return sum((particle.accepted_sum_stats + particle.rejected_sum_stats
                     for particle in self._particles), [])
 
+    def first_n_sum_stats(self, n):
```

Comment:
Note that a particle.accepted_sum_stats list can contain >1 entries. So the number might be >n here actually. But this is an old feature of pyabc which I am not aware of anyone using (and will probably be deleted soon), so I guess this should be fine.